openwisp_radius/models.py

```diff
@@ -103,3 +106,13 @@ def _get_instance(self, **kwargs):
     class Meta(AbstractRadiusUserProfile.Meta):
         abstract = False
         swappable = swappable_setting('openwisp_radius', 'RadiusUserProfile')
+
+
+class OrganizationRadiusSettings(OrgMixin, models.Model):
+    id = models.UUIDField(default=uuid.uuid4, primary_key=True)
+    token = models.CharField(max_length=15)
```

Comment:
can you add a default value which is automatically generated and a validator like we do in openwisp-controller?
https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/models.py#L218-L219
You can copy that code, then we can create some GCI dedicated issues to move that logic into openwisp-utils.

Reply:
@nemesisdesign the default length of netjsonconfig's ``get_random_key`` is 32. Should we use https://github.com/django/django/blob/master/django/utils/crypto.py#L48 this instead? We can set it to 15, and I can write a new validator, or it can be set as a GCI task.